app/src/main/java/com/example/easysurvey/ui/theme/MainActivity.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.easysurvey.ui.theme
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.ui.Modifier
+
+class MainActivity : ComponentActivity(){
+    private var viewModel: MainViewModel by viewModel<MainViewModel>()
+    private var Anwser: String = ""
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent {
+            viewModel.fetchSurveys()
+
+        }
+    }
+}
```

Comment:
I added the main activity class as specified in class video

Reply:
This looks like an incomplete feature.
Are you using these anywhere?  I don't see any composables called in the setContent{}
If not in use, remove them.  Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.